app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -434,14 +437,17 @@ function handleSizeChange(event) {
 
 function handleButtonsActivation(event) {
   var textarea = event.target;
-  var buttons = textarea.closest('.comment-form').getElementsByClassName('js-btn-enable');
-  Array.from(buttons).forEach(function(button) {
-    if (textarea.value.length > 0) {
-      button.disabled = false;
-    } else {
-      button.disabled = true;
-    }
-  });
+  var commentForm = textarea.closest('.comment-form');
+  if (commentForm) {
+    var buttons = textarea.closest('.comment-form').getElementsByClassName('js-btn-enable');
+    Array.from(buttons).forEach(function(button) {
+      if (textarea.value.length > 0) {
+        button.disabled = false;
+      } else {
+        button.disabled = true;
+      }
```

Comment:
Refactor (blocking): This can be more terse while still being readable.

```suggestion
    button.disabled = textarea.value.length === 0
```

Reply:
Wasn't sure how much to refactor this as it's not super related to this PR but yep lets go for it 🙂